src/scss/_base.scss

```diff
@@ -36,21 +36,17 @@
 		text-align: left;
 		vertical-align: top;
 	}
-	
+
 	.primary-data {
```

Comment:
If referring to `.primary-data` and `.secondary-data`, then it was suggested between me, @SarahHornsey and @rowanmanning so that it was obvious for the usecase for other product teams to use instead of using a BEM version.

I don't think `.primary-data` and `.secondary-data` will get a lot of use but it is there in case product teams want to use it.